packages/web-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,121 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+} from '../theme/design-tokens.js';
+
+/** Radio styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-grid')}
+
+  :host {
+    grid-auto-flow: column;
+    grid-template-columns: max-content;
+    gap: ${spacingHorizontalXS};
+    align-items: center;
+    height: 32px;
+    cursor: pointer;
+    outline: none;
+    position: relative;
+    user-select: none;
+    color: var(--state-color);
+  }
+  :host([disabled]) {
+    color: ${colorNeutralForegroundDisabled};
+    pointer-events: none;
+  }
+  .label {
+    cursor: pointer;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
```

Comment:
Just a note, these both do the same thing in browsers with the exception (IIRC) that you can transition visibility but not display.